ci-operator/step-registry/fips-check/fips-check-commands.sh

```diff
@@ -44,3 +51,5 @@ for (( i=0; i<$num_nodes; i++ )); do
     fi
   fi
 done
+
+oc delete ns "$TEST_NS"
```

Comment:
Might be useful to disable `set -o errexit`? Or at least for parts of the script?

Reply:
let's keep the changes minimal